sdk/storage/Azure.Storage.Common/src/UploadTransactionalHashingOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage
```

Comment:
Did the Arch Board decide on `Azure.Storage` instead of `Azure.Storage.Common`?

Reply:
They did.